.github/ISSUE_TEMPLATE/requirement.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Requirement
+about: Use this template to propose an LWS requirement.
+title: "[REQ] <<brief description of requirement>>"
```

Comment:
Should we have two "sub-tags" for functional / non-functional requirements?
e.q. [REQ-F] / [REQ-NF]

Reply:
An atomic UC requirement is typically either functional or non-functional, so it's best to express it as either a REQ-F or a REQ-NF (without mentioning abstract REQ).

Reply:
Done